Lombiq.HelpfulLibraries.sln

```diff
@@ -0,0 +1,40 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30114.105
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Lombiq.HelpfulLibraries", "Lombiq.HelpfulLibraries\Lombiq.HelpfulLibraries.csproj", "{5F3379C7-ED6B-46AA-B6C3-17CBC2284417}"

```

Comment:
I had to add a solution, to be able to publish all without specifying by project name

Reply:
This is good, just please document its purpose to the Readme.

Reply:
Done (please edit if not what you wanted)